xml/resources-2.8.rng

```diff
@@ -65,14 +65,14 @@
     </element>
   </define>
 
-
   <define name="element-bundle">
     <element name="bundle">
-      <attribute name="id"><data type="ID"/></attribute>
-      <optional>
-        <attribute name="description"><text/></attribute>
-      </optional>
       <interleave>
+        <attribute name="id"><data type="ID"/></attribute>
+        <optional>
+          <attribute name="description"><text/></attribute>
+        </optional>
+        <ref name="element-resource-extra"/>
```

Comment:
I was more thinking that we would enumerate the different options we need rather than go the nvpair route.
But your call.